### ISSUE TITLE
TinyMCE editor missing in Beaver Builder

### ISSUE THREAD

**Issue by viktorix** [ClassicPress-research/ClassicPress-Editor#13]
It looks like the editor is missing inside a text module, but there are no console errors:
![image](https://user-images.githubusercontent.com/1692600/131281178-64afde50-28cc-4921-ab29-c8417b774151.png)

This is ClassicPress 1.3.0 RC2,  Beaver Builder Lite 2.4.2.5, and ClassicPress Editor 1.0.0.

**Comment by ClassyBot** [comment]
This issue has been mentioned on **ClassicPress Forums**. There might be relevant details there:

https://forums.classicpress.net/t/tinymce-v5-support-in-beaver-builder/3500/1

**Comment by ClassyBot** [comment]
This issue has been mentioned on **ClassicPress Forums**. There might be relevant details there:

https://forums.classicpress.net/t/tinymce-v5-upgrade-and-backwards-compatibility/3515/12

**Comment by Pross** [comment]
Your plugin only enqueues the tiny5 js/css when in wp-admin and on certain pages so when BB is enabled on the frontend none of the tiny code is loaded so the BB text editor falls back to text mode.

To get tiny5 to work with BB we will have to enqueue the tiny scripts when the BB UI is open, I tested this already and I got the editor to init fine tho our code will have to be modded to hide it until its needed, not a big deal I don't think. I'll be following progress and when its added to core I'll make the changes needed to BB to make use of it.
